tests/docker/Dockerfile

```diff
@@ -196,6 +196,24 @@ RUN /keycloak && rm /keycloak
 
 #################################
 
+FROM golang as tinygo
+
+COPY --chown=0:0 --chmod=0755 tests/docker/ducktape-deps/tinygo /
+RUN /tinygo && \
+    rm /tinygo
+ENV PATH="${PATH}:/usr/local/tinygo/bin"
+
+#################################
+
+FROM tinygo as wasi-transforms
+
+COPY --chown=0:0 --chmod=0755 tests/docker/ducktape-deps/tinygo-wasi-transforms /
+COPY --chown=0:0 --chmod=0755 src/go/transform-sdk /transform-sdk
+RUN /tinygo-wasi-transforms && \
+    rm -rf /transform-sdk
+
+#################################
+
```

Comment:
This looks right but it's also been a while since I've added deps to Ducktape. Did you ask devprod if this is the correct location for deps?

Reply:
Confirmed that this is the right spot.